fedmsg/consumers/ircbot.py

```diff
@@ -307,9 +313,10 @@ def prettify(self, topic, msg, pretty=False, terse=False, short=False):
                     link = fedmsg.meta.msg2link(msg, **self.hub.config)
 
                 if link and short:
-                    dagd = 'http://da.gd/s'
-                    resp = requests.get(dagd, params=dict(url=link))
-                    link = resp.text.strip()
+                    if callable(short):
```

Comment:
Is the intention here to use some other shortener service? I ask because this approach is probably going to make error handling tricky since any sort of exception could be raised if someone opts to not use requests.